src/ec.rs

```diff
@@ -326,7 +326,6 @@ impl<S> WriterBase<S> {
         debug: std::env::var_os("RAV1E_DEBUG").is_some(),
         fake_bits_frac: 0,
         s: storage,
-        cpu,
```

Comment:
?

Reply:
another problem fixed while testing the patch.